### ISSUE TITLE
Fix aarch64 build in the CI

### ISSUE THREAD

**Issue by JohanMabille** [xtensor-stack/xsimd#410]
The build on linux aarch64 is currently disabled in the CI (because it fails) since #409 has been merged. It should be fixed and enabled again.

**Comment by pitrou** [comment]
(note that Travis-CI provides native Arm64 workers)

**Comment by JohanMabille** [comment]
Indeed, we use them before. We switched from travisci on purpose (I don't want to beg travisci for build time for each project of the stack;)).

**Comment by Alex-PLACET** [comment]
Can we close this issue as the aarch64 compilation is green ?
